modules/support-collecting-network-trace.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+[id="support-collecting-network-trace_{context}"]
```

Comment:
https://github.com/openshift/must-gather/pull/156 recently merged it might be work calling this out too. 
@pecameron  for review.

Reply:
That is a good idea, thanks for highlighting this. I have added a to do item in the module in future.